go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	golang.org/x/sys v0.11.0
 	google.golang.org/grpc v1.57.0
 	google.golang.org/protobuf v1.31.0
+	software.sslmate.com/src/go-pkcs12 v0.2.1
```

Comment:
I guess we'll need to update the third party license again, right?

Reply:
yes, I am planning to have a follow up PR for updating the licenses after this.